components/governance/ProposalDescription.vue

```diff
@@ -6,13 +6,7 @@
     >
       <h4>Description</h4>
       <div class="description">
-        <pre
-          v-if="
-            proposal.type === `PARAMETER_CHANGE` || proposal.type === `TREASURY`
-          "
-          >{{ proposal.description }}</pre
-        >
-        <p v-else>{{ proposal.description }}</p>
+        <pre>{{ proposal.description }}</pre>
```

Comment:
sure? there was a purpose in not using a pre I remember

Reply:
yeah it's ok. plaintext proposals might be "nicer" without a `pre` wrapper but it's not really a big deal and i didn't feel like making sure all the styles worked properly for `p` tags - so `pre` is ok for now.

Reply:
lazy...